packages/core/src/index.ts

```diff
@@ -164,7 +189,23 @@ export function apply(ctx: Context, config: Config) {
       if (!filtered?.length) return session?.text('.no-result')
 
       const output: (string | Element)[] = []
+
       for (const image of filtered) {
+        if (config.asset && ctx.assets) {
+          const temp = await ctx.booru.imgUrlToAssetUrl(image)
+          if (!temp?.length) {
+            output.unshift(session.text('.no-image'))
+            continue
+          }
+          image.url = 'http://' + temp
```

Comment:
Why concatenate string here. Will it work with other assets like `s3` or `smms` or something?

Reply:
`temp` is an URL without protocol header.

Reply:
Then how about `s3` or `smms` or others?